recipes/xalan-c/meta.yaml

```diff
@@ -0,0 +1,66 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "xalan-c" %}
+{% set version = "1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # xalan lib uses <major>_<minor> in links intead of <major>.<minor>
+  url: https://github.com/apache/xalan-c/archive/Xalan-C_{{ version|replace(".","_") }}_0.tar.gz
+  sha256: edfaee4562d9dec4d0a971e06cc6355f8c822ce49f18beba095f8a4dc93b4285
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - xerces-c >=3
+  run:
+    - xerces-c >=3
+
+test:
+  commands:
+    - which Xalan
+
+about:
+  home: https://github.com/simplejson/simplejson
+  license: Apache-2.0
+  license_file: LICENSE.txt
```

Comment:
There is an in built ability in conda-build to look for this file in the provided source. Since the license is already part of the archive, you don't need to place it here in the recipe. Therefore, you should be able to replace `LICENSE.txt` with `LICENSE` here.